// js-swiper/dist/cjs/src/lib/styles/styled.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Image = exports.ImageWrapperA = exports.ImageWrapperDiv = exports.ImagesWrapper = exports.TimerBar = exports.Container = void 0;
const styled_1 = require("@emotion/styled");
exports.Container = styled_1.default.div `
  position: relative;
  display: flex;
  flex-direction: column;
  width: 100%;
  height: 100%;
  overflow: hidden;
  user-select: none;
  -webkit-touch-callout: none;
`;
exports.TimerBar = styled_1.default.div `
  position: absolute;
  bottom: 0;
  left: 0;
  flex: 0 0 4px;
  width: 100%;
  height: 4px;
  background-color: rgba(0, 0, 0, 0.2);
  z-index: 2;
`;
exports.ImagesWrapper = styled_1.default.div `
  flex: 0 0 100%;
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
  width: 100%;
  height: 100%;
`;
exports.ImageWrapperDiv = styled_1.default.div `
  display: flex;
  width: 100%;
  height: 100%;
  background-color: ${props => props.theme.backgroundColor}
`;
exports.ImageWrapperA = styled_1.default.a `
  display: flex;
  width: 100%;
  height: 100%;
  background-color: ${props => props.theme.backgroundColor}
`;
exports.Image = styled_1.default.img `
  width: 100%;
  height: 100%;
  object-fit: ${props => { var _a; return (_a = props.theme.objectFit) !== null && _a !== void 0 ? _a : 'cover'; }};
  object-position: center;
`;
